docs/build/voting-dapp-tutorial.md

```diff
@@ -0,0 +1,1245 @@
+---
+title: dApp Tutorial
+description: Build a complete dApp on Kadena
+---
+
+import PageRef from '@components/PageRef'
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Complete dApp Tutorial
+
+Build a complete on-chain voting dApp
+
+---
+
+## Overview
+
+In this tutorial you will build a complete dApp, smart contracts and frontend that allows users to submit a vote. You will also deploy your own gas station that will be used to pay for the gas needed to interact with your dApp.
+
+By leveraging the blockchain and smart contracts to implement this application, its users benefit of complete transparency of the voting process, anyone being able to check every vote that was recorded, as well as peace of mind that their vote will never be deleted. This set of features is simply not possible in a classic Web2 application.
+
+## Environment Setup
+
+### Install Pact
+
+<Tabs>
+  <TabItem value="mac" label="Mac">
+
+    brew update
+    brew install kadena-io/pact/pact
+  </TabItem>
+
+  <TabItem value="linux" label="Linux">
+    For installing Pact on Linux distributions in the Arch family, refer to <a href="https://aur.archlinux.org/packages/pact/">this package on the AUR</a>. Otherwise, please refer to <a href="https://github.com/kadena-io/pact#building-from-source">building from source</a>.
```

Comment:
We haven't updated that AUR package in forever. Let's maybe copy the walk through of installing via `nix` instead (and also link to the `buildFromSrc`

Reply:
Updated. And I just found out there's a Wiki.